src/screens/Departures/AllStopsOverview.tsx

```diff
@@ -216,7 +175,6 @@ const PlacesOverview: React.FC<PlacesOverviewProps> = ({
     <SimpleDisappearingHeader
       onRefresh={refresh}
       isRefreshing={isLoading}
-      leftButton={leftButton}
```

Comment:
Should it no longer show icon on service disruption? Is this intended or just that the icon is not updated in the design?

Reply:
Ah. I see it is used in the tab navigator instead. Should this then actually be `SimpleDisappearingHeader`? I get it is easy to use to refresh, but when removing header title and left button here the idea of DisappearingHeader goes away and it is now more used like a FlatList with pull to refresh? I think there might be better components for this instead of in a way deluding the concept of `SimpleDisappearingHeader`

Reply:
Testing it more now and seeing real use. Scratch this comment. It makes sense with favorites and input box

Reply:
Yes, I started with not using SimpleDisappearingHeader first but realized that it was doing a lot behind the scenes. So kept it as-is.